### PR TITLE
 [ASoC 2022] Pixiu Metrics Implementation #433

### PR DESCRIPTION
**Pull request by gongna-au**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #433 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/metrics/global/global.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package global
+
+const (
+	ListenAddress  = ":9114"
+	MetricsPath    = "/metrics"
+	URI            = "http://localhost:9200"
```
Comment (PhilYue) [inline pkg/metrics/global/global.go]: 配置别写死了，放到配置文件，看看和现有的配置能否合并一下 https://github.com/apache/dubbo-go-pixiu/blob/develop/pkg/model/metric.go

Comment (gongna-au) [reply pkg/metrics/global/global.go]: ok

File: pkg/metrics/metic_manager.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+		return nil
+	}
+}
+
+type MeticManager struct {
+	driver    *MeticDriver
+	bootstrap *model.Bootstrap
+}
+
+func CreateDefaultMeticManager(bs *model.Bootstrap) *MeticManager {
```
Comment (PhilYue) [inline pkg/metrics/metic_manager.go]: pixiu 什么时候创建呢

File: pkg/metrics/metic_manager.go [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+	if bs.Metric == st {
+		logger.Info("[dubbo-go-pixiu] no metric configuration in conf.yaml")
+		return nil
+	} else {
+		driver.cfg = bs.Metric
+	}
+	return driver
+}
+
+func (d *MeticDriver) SetMetricCollector(hc *http.Client, loger logger.Logger) error {
```
Comment (PhilYue) [inline pkg/metrics/metic_manager.go]: pixiu 什么时候触发 register collector

**Pull request closed without merging**